device-types/Cisco/C9105AXI-E.yaml

```diff
@@ -0,0 +1,20 @@
+---
+manufacturer: Cisco
+model: Catalyst 9105AXI-E
```

Comment:
Should include all variants of the 9105AXI (-A, -B, -H, -Q, -K, -T, -G, -D, -S, -I, -Z, -F, -N, -C, -R) if we are specifying the "-E".